Source/d_loop.c

```diff
@@ -721,7 +725,7 @@ void TryRunTics (void)
 
         // [AM] If we've uncapped the framerate and there are no tics
         //      to run, return early instead of waiting around.
-        if (uncapped && realtics == 0 && leveltime > oldleveltime)
```

Comment:
Or maybe just `realtics <= 0` if something is indeed flawed with its calculation?

Reply:
Due to this lines interpolation don't work in complevel boom after the level change: https://github.com/fabiangreffrath/woof/blob/bf31a22f5fa9402b330c928d55dc6c053c5913de/Source/g_game.c#L583-L584

Also, I don't like this fix, I should research more.